astropy/coordinates/sky_coordinate.py

```diff
@@ -1003,6 +1007,105 @@ def to_string(self, style='decimal', **kwargs):
 
         return coord_string
 
+    def to_table(self, table=None, column_conflicts='warn',
+                 metadata_conflicts='warn'):
+        """
+        Convert the data in this |SkyCoord| to a |QTable|.
+
+        Any attributes that have the same length as the |SkyCoord| will be
+        columns in the output. All other attributes will be recorded as
+        metadata.
+
+        If ``table`` is a |Table| whose length matches that of the |SkyCoord|
+        then the columns and metadata of the converted |SkyCoord| are merged
+        with it. In case of a column name or metadata conflict data from the
+        |SkyCoord| is preferred.
+
+        Parameters
+        ----------
+        table : |Table|, optional
+            A table to which the data from the |SkyCoord| is added.
+        column_conflicts : str ('warn'|'error'|'silent')
+            How to handle column name conflicts. Has no effect if ``table`` is
+            not provided.
+        metadata_conflicts : str ('warn'|'error'|'silent')
+            How to handle metadata conflicts. Has no effect if ``table`` is not
+            provided.
+
+        Returns
+        -------
+        |QTable|
```

Comment:
For the return type it is preferred to be explicit
```suggestion
        `~astropy.table.QTable`
```

Reply:
If that's the convention then it's best to adhere to it.